### PR TITLE
ENTESB-16374 Fail to install jaeger-operator on openshift3

### PR DESCRIPTION
**Pull request by claudio4j**
Re-generated the assets_vfsdata.go related to a previous change in 03-syndesis-server-config.yml.tmpl
https://issues.redhat.com/browse/ENTESB-16374

### CODE REVIEW

**Comment by avano** [comment]
retest this please

File: install/operator/pkg/syndesis/capabilities/capabilities.go [hunk 1]
```diff
@@ -37,15 +37,15 @@ type RequiredApiSpec struct {
 	routes                    string
 	imagestreams              string
 	oauthclientauthorizations string
-	catalogsources            string
+	packagemanifests          string
```
Comment (zregvart) [inline install/operator/pkg/syndesis/capabilities/capabilities.go:40]: I'm assuming this is part of this PR?

**Review by zregvart — APPROVED** [review]
Thanks!

File: install/operator/pkg/syndesis/capabilities/capabilities.go (hunk 1 above)
Comment (zregvart) [reply install/operator/pkg/syndesis/capabilities/capabilities.go:40]: Looks like it needs to be.

Comment (claudio4j) [reply install/operator/pkg/syndesis/capabilities/capabilities.go:40]: Yes, I replaced `catalogsources` for `packagemanifests` to test if openshift can install from OLM. At least in my openshift 3.11, `catalogsources` does exist but lacks OLM operator and other OLM objetcs. 
Therefore, I found that `packagemanifests` better represents a CR to enable operators to install from operatorhub.

**Pull request merged**